.dockerignore

```diff
@@ -0,0 +1,5 @@
+.git
+.dockerignore
+Dockerfile
+CONTRIBUTORS.md
+README.md
```

Comment:
It might make sense to add `docker-build.sh` here as that's only ever used outside of the container.

Reply:
Yes indeed, done.